frontend/src/pages/RecipientRecord/pages/CommunicationLogForm/pages/log.js

```diff
@@ -43,6 +49,25 @@ const Log = ({ datePickerKey }) => {
         </ReadOnlyField>
       </div>
 
+      <div className="margin-top-2">
+        <FormItem
```

Comment:
This field doesn't validate on blur, should it?

Reply:
thanks, it should now: eff97b9d2